pkg/reconciler/v1alpha1/revision/cruds.go

```diff
@@ -47,8 +48,7 @@ func (c *Reconciler) createDeployment(ctx context.Context, rev *v1alpha1.Revisio
 	return c.KubeClientSet.AppsV1().Deployments(deployment.Namespace).Create(deployment)
 }
 
-func (c *Reconciler) checkAndUpdateDeployment(ctx context.Context, rev *v1alpha1.Revision, have *appsv1.Deployment) (*appsv1.Deployment, Changed, error) {
-	logger := logging.FromContext(ctx)
```

Comment:
I'd leave this as-is.  At a minimum you've inconsistently changed this.

Reply:
the logger that was here was not anntotated witht he deployment, the passed in does. I can create from context and annotate, but now we have duplicated same code twice and created two more objects.

Reply:
I personally think we went a little crazy with the logger decoration.  We already put the revision name on there and the deployment is a straight function of that 🤷‍♂️ 

Personally I'd still bias towards leaving this is and getting rid of the gratuitous logger decoration :)

Reply:
Done.